$NetBSD: patch-config_manifest.inc.php,v 1.1 2025/03/06 01:21:38 manu Exp $

Update hashes and size of files where we substituted interpreters

--- config/manifest.inc.php.orig	2025-03-05 01:41:14.877268215 +0100
+++ config/manifest.inc.php	2025-03-05 01:46:26.982551207 +0100
@@ -15,9 +15,9 @@
 		"config/environment/dev.php" => array("476", "7b39e3f4249647a41d5de066ffca49ba42c34c13df53e22372e4861d0334ce47"),
 		"config/environment/ui-test-tidb.php" => array("496", "e00f296d72ae40cae1b96fa3d2ad5e0c6eb5b3871b35ff3f57e8a3d2f6f41e87"),
 		"config/global.ini.php" => array("71103", "efe79268b295ef7aedb63a10b64b779cbd895908eede301c62f8967909b4c61b"),
 		"config/global.php" => array("10128", "1d07a02e8207a9ba9d13e1122b1e8594456175c43f0612f2e107b04783a44333"),
-		"console" => array("753", "737cdb183502eed206d23a39ec262675aa53c6de4d9208c73c84bff8ea82aef8"),
+		"console" => array("753", "ba2702633ca4b127dd4de737f70c4f3a51e8275b9403aa2d36644034717b7200"),
 		"core/API/ApiRenderer.php" => array("4023", "620a28e1f148b894784a90803908cf09f20c54586356e3665cc5b7bc54dd9cdc"),
 		"core/API/CORSHandler.php" => array("1435", "d5c4d8b5cf2c0d22900cdbbab20f466a806803efdd0402a1447a0d794188a337"),
 		"core/API/DataTableGenericFilter.php" => array("7602", "3fcf08862fa7a1cd6fd644e19b420d316f408a8d3da421bf1afeee1430f1ebfe"),
 		"core/API/DataTableManipulator.php" => array("7540", "6cb6c60411ea27017516a76f1c7ded089ace86a13a0e736bb796cb2e54557ee0"),
@@ -947,9 +947,9 @@
 		"misc/log-analytics/CONTRIBUTING.md" => array("349", "5386a3709b60d999e5ac55c8995869c95c464561c57585d16fa790adf70f1fcb"),
 		"misc/log-analytics/LEGALNOTICE" => array("673", "618af57dfdc425c22ec0c2907f833dc96b264220cf8b45cbe668af652d04cacb"),
 		"misc/log-analytics/LICENSE.txt" => array("35133", "ecc0f61bb4572d9915f078ccd67244140040e67b3d3be59cd04851cc7eff7b1c"),
 		"misc/log-analytics/README.md" => array("19858", "3260ebf7008bd9ffcbb1d0b8d515e5571fbe5ef6567d744c1f7a3a2187c80ac1"),
-		"misc/log-analytics/import_logs.py" => array("111737", "b25f72305d55f8c09e6d1e6aaf335938ffa0ad54c86cc1f6311d493dc3976b36"),
+		"misc/log-analytics/import_logs.py" => array("111745", "6706d48bf96f995865620a5805483072a8e01c49c25e4ac54a3237027ef6b81d"),
 		"misc/others/ExampleMatomoTracker.php" => array("674", "7e9e7c5dfdc29f3a7707fbb6287d941843dd1579bafa823921ed68b81076df20"),
 		"misc/others/geoipUpdateRows.php" => array("304", "78169f84391c88705067a97f6c4f9f187677d4b0986be54c282dcf1540eedec0"),
 		"misc/others/tracker_simpleImageTracker.php" => array("1053", "05366d92d3eb46428098140bf319bdbaef49ef794868fd6a43a33eca7bcfbb78"),
 		"misc/others/uninstall-delete-matomo-directory.php" => array("1242", "812fa5080bb6e4f907e6be6281aea383b513b80c460daf2dba4bf33d925d039b"),
@@ -3309,9 +3309,9 @@
 		"plugins/GeoIp2/Controller.php" => array("7942", "92677dc1e7e892b9c8175c7d4a3730d6f4e90c8ee8a67a90e5426def5a8ea33a"),
 		"plugins/GeoIp2/GeoIP2AutoUpdater.php" => array("30509", "a1336a57af104aa4cdcd60ea2a505c52ac7fb42ddc6daadd452f7ff54a67d7bb"),
 		"plugins/GeoIp2/GeoIp2.php" => array("5161", "c320b96984a1341bc9c4a9211fd75a284bb6272d4caa7842d1828225018a83d1"),
 		"plugins/GeoIp2/LocationProvider/GeoIp2.php" => array("8218", "e7e0d568d4f92329c0fc244918f5fdb092d1e0aa9145a88748403cc3dc30fa78"),
-		"plugins/GeoIp2/LocationProvider/GeoIp2/Php.php" => array("21887", "a280b569610b829560f33ab77265e7630aaf2b2fe78c66066c3ab20cece17a30"),
+		"plugins/GeoIp2/LocationProvider/GeoIp2/Php.php" => array("21867", "61ee5c05f5386ea126b12b2ecb17467f132abdcb6391c3ae94782e3bd9fe3b1e"),
 		"plugins/GeoIp2/LocationProvider/GeoIp2/ServerModule.php" => array("13166", "74dc53f99712e61aff0a5118c42a9ca862b8108b0ebb4538a0460d0925152047"),
 		"plugins/GeoIp2/SystemSettings.php" => array("1950", "78326d36d307ec21b471b6b361cbbbaf5740319112ab271a931d2ab7c3b21ae9"),
 		"plugins/GeoIp2/Tasks.php" => array("585", "b45571ef0bacc6722cc05221aebb0e4b1d2774446e3a760d6202bf0389568055"),
 		"plugins/GeoIp2/config/config.php" => array("112", "fe0ffa6f97f8f3844fca559d1e0aab5b8e2dc1804f7cb96b07accc470f67b8aa"),
@@ -10181,10 +10181,10 @@
 		"vendor/symfony/error-handler/Resources/assets/images/icon-support.svg" => array("634", "dc18aace119f62ae4c0e90356483aff5831bc841aaa128305b1c83aaed843aec"),
 		"vendor/symfony/error-handler/Resources/assets/images/symfony-ghost.svg.php" => array("8180", "8270b787efa507721636e378057192c646750182a2ab8085148ea09b24ce4fa2"),
 		"vendor/symfony/error-handler/Resources/assets/images/symfony-logo.svg" => array("942", "24fde082658c2ceace638353000edfc84b0a4b0581235dc4bb0063e50de81674"),
 		"vendor/symfony/error-handler/Resources/assets/js/exception.js" => array("13343", "2bbdf4b696f364258e12ab59ff8df245a6bb1e7cd67d581416b2546391addc50"),
-		"vendor/symfony/error-handler/Resources/bin/extract-tentative-return-types.php" => array("2027", "7bbb102a2f26e04e4fa83273fb47349246e552323e5639345bc953642453a9db"),
-		"vendor/symfony/error-handler/Resources/bin/patch-type-declarations" => array("3705", "98e3d6db6ccfcd6713729bc4d8192bfe0d3ce05f89a86c9dca4b3007cf7b4510"),
+		"vendor/symfony/error-handler/Resources/bin/extract-tentative-return-types.php" => array("2027", "da070a0fb40564fd9081fc7469dbbb4f297351efb454c5f5f33adace3aedaaa1"),
+		"vendor/symfony/error-handler/Resources/bin/patch-type-declarations" => array("3705", "7499d3e284a85494e58e85fd64117793f4d8ee692c44619322cd4da893c9aaf4"),
 		"vendor/symfony/error-handler/Resources/views/error.html.php" => array("643", "fc116c4e624911d9fec21cc9114164d58267bddc36e47c439043e59a928311dd"),
 		"vendor/symfony/error-handler/Resources/views/exception.html.php" => array("4542", "ac0264526c021365d89bbbac522acb91554c34fae61987a48747e7820b566990"),
 		"vendor/symfony/error-handler/Resources/views/exception_full.html.php" => array("1841", "0288256c0d6a8b072c4c72c857212177631502d1fec5ed642ad366c795d2f292"),
 		"vendor/symfony/error-handler/Resources/views/logs.html.php" => array("2139", "1f8b724c0022c99b938e5bba3641a342cac82e252e48331a68e7989212c580f5"),
@@ -10704,9 +10704,9 @@
 		"vendor/symfony/var-dumper/Dumper/ServerDumper.php" => array("1606", "7933c15190aaf74fab75ace662f23dd49db8f0d3fa57b05c27bca5a6e74525cb"),
 		"vendor/symfony/var-dumper/Exception/ThrowingCasterException.php" => array("655", "f62bf45b0b53c4184c26818a9d340c641c381c39f99d7ce6b799cd536005f46d"),
 		"vendor/symfony/var-dumper/LICENSE" => array("1068", "cbad2ab8fcfa03932eaef0d2c88482f30a4198f18ae7ed6f9cef96a94e861296"),
 		"vendor/symfony/var-dumper/README.md" => array("607", "34283089fc922d6ac7447a142a258c4ccd0814c67b82968a1a8b0907a2f709e7"),
-		"vendor/symfony/var-dumper/Resources/bin/var-dump-server" => array("2142", "aae76491c4ba082ede3dc478831c1f1f5edb3e6c46785fc7b25023520f8cd5f6"),
+		"vendor/symfony/var-dumper/Resources/bin/var-dump-server" => array("2142", "991a2b654ee06ade5381fd28638c062ef39737363b28fb3a8385a43116fab4ca"),
 		"vendor/symfony/var-dumper/Resources/css/htmlDescriptor.css" => array("3010", "d6228cb909961e9521fe39dc928d42c875a2abcf7d244e1ce53e7c8b3718060e"),
 		"vendor/symfony/var-dumper/Resources/functions/dump.php" => array("993", "1d34624594296b06e5cd4423ad39f050265e1c3bba1ae63d6d02529bd5eda587"),
 		"vendor/symfony/var-dumper/Resources/js/htmlDescriptor.js" => array("354", "f2f5ed0b97d9084213608a5583b78410403eb4ffdbbc49da61e4e0a2b3c41be4"),
 		"vendor/symfony/var-dumper/Server/Connection.php" => array("2721", "bb2fdd5e39fe29841e48823049ccc7a9ef2179f7ae3caa3f3757129ec2bbeaa9"),
@@ -10993,9 +10993,9 @@
 		"vendor/wikimedia/less.php/LICENSE" => array("11358", "cfc7749b96f63bd31c3c42b5c471bf756814053e847c10f3eb003417bc523d30"),
 		"vendor/wikimedia/less.php/NOTICE.txt" => array("806", "6dcd8c4cb45e00ce710f41e6a62f5122484b602535223d52bf658b1c664282ff"),
 		"vendor/wikimedia/less.php/README.md" => array("4596", "f1dbdef17c8f011927ca9b3eb9230b8cbefc615ccb792cdf82d300178abd7145"),
 		"vendor/wikimedia/less.php/SECURITY.md" => array("213", "726700674fc04dda2be4131f39e2b1f9de189b190fc474366223bd29c3a4ab17"),
-		"vendor/wikimedia/less.php/bin/lessc" => array("4034", "d0bd5904ec5fb0985a55a8035d2fe4b2876ca7e31a710795a8c420d370620fad"),
+		"vendor/wikimedia/less.php/bin/lessc" => array("4034", "740b6ffd4563633b8b4142a033ed4ea6bf27b7d1057a6af970439fe428ad0b85"),
 		"vendor/wikimedia/less.php/lessc.inc.php" => array("7185", "6ad74675960b0d996a23da1a8b37b7dcec2d43a868c557f1473e5809dd87bcc3"),
 		"vendor/wikimedia/less.php/lib/Less/.easymin/ignore_prefixes" => array("24", "c50b79f9741f7d49fe76033bd3758a475c9005c1de103490c62571aa5a85360b"),
 		"vendor/wikimedia/less.php/lib/Less/Autoloader.php" => array("1160", "de71130fcd7e15290fdb9378570badce046b70acd24e6e6ecd045e62ce8cbe99"),
 		"vendor/wikimedia/less.php/lib/Less/Cache.php" => array("7720", "3dad40128ac5bd460b57fa5271e0ef23a1e0a72fdf5c0d03c43a505237b42b66"),
